apps/www/pages/blog/[slug].tsx

```diff
@@ -55,7 +107,7 @@ export async function getStaticProps({ params }: any) {
       nextPost: currentIndex === allPosts.length ? null : nextPost ? nextPost : null,
       relatedPosts,
       blog: {
-        slug: `${params.slug}`,
+        slug: `${params?.slug}`,
```

Comment:
Since we know `params` should be defined by now:
```suggestion
        slug: `${params!.slug}`,
```

Reply:
This can be removed entirely I believe, as `params?.slug === undefined` discriminate the type.

https://www.typescriptlang.org/play?#code/GYVwdgxgLglg9mABMOcAUBvRAHAhgJ1wFsBnRAXwC5Es9DSB+arEgGxAHMnESp8YwHChQCUNAFCIpiGMERo6xEgwB0bTogC82xOAAmAU2ACDesRknSr+A1BD4wlqeXFPEEBL0QB3XFC04BEpq7BxuAPThVgB6DOLkQA

Reply:
Yep! My bad, I mistakenly thought we were in another scope (eg. `map()`). Good catch.